manage/trigger/how-do-they-work.rst

```diff
@@ -111,3 +111,25 @@ A trigger can do the following things once its conditions have been met:
    This allows you to help your agents with specific information if needed.
    (e.g. automated changes a trigger applied to the ticket)
 
+Localization of Trigger Based Notifications
+-------------------------------------------
+
+To translate the output of variables and to format date and time stamps for
+notifications in triggers, use the ``t`` and ``dt`` methods.
```

Comment:
"To translate the output of variables, use the ``t`` function. To format date and time stamps, use the ``dt`` function."